cmd/init.go

```diff
@@ -1,58 +1,59 @@
 package cmd
 
 import (
-	"github.com/spf13/cobra"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
 
 	"github.com/cmoulliard/k8s-supervisor/pkg/common/oc"
 
 	"github.com/cmoulliard/k8s-supervisor/pkg/buildpack"
 )
 
 var initCmd = &cobra.Command{
-	Use:   "init [flags]",
-	Short: "Create a development's pod for the component",
-	Long:  `Create a development's pod for the component.`,
+	Use:     "init [flags]",
+	Short:   "Create a development's pod for the component",
+	Long:    `Create a development's pod for the component.`,
 	Example: ` sb init -n bootapp`,
-	Args: cobra.RangeArgs(0, 1),
+	Args:    cobra.RangeArgs(0, 1),
 	Run: func(cmd *cobra.Command, args []string) {
 
 		log.Info("Init command called")
-		log.Debug("Namespace : ", namespace)
+		log.Debugf("Namespace: %s", namespace)
 
-		// Parse MANIFEST - Step 1
+		// Parse MANIFEST
 		application := parseManifest()
 
-		// Add Namespace's value
-		application.Namespace = namespace
-
-		// Get K8s' config file - Step 2
+		// Get K8s' config file
 		kubeCfg := getK8Config(*cmd)
 
-		// Execute oc command to switch to the namespace defined
-		log.Info("[Step 3] - Get k8s default's namespace")
-		oc.ExecCommand(oc.Command{Args: []string{"project",application.Namespace}})
+		// Switch to namespace if specified or retrieve the current one if not
+		currentNs, err := oc.ExecCommand(oc.Command{Args: []string{"project", "-q", namespace}})
+		if err != nil {
+			log.Fatal(err)
+		}
+		log.Infof("Using '%s' namespace", currentNs)
+		application.Namespace = currentNs
 
-		// Create Kube Rest's Config Client - Step 3
```

Comment:
Why do you prefer to remove "-STEP" text ? @metacosm

Reply:
Because they don't appear consistently and I find it confusing…  For example, I never saw Step 1 or Step 2 with `init`

Reply:
That could be hard to maintain as that will evolve over the time. I will merge the PR then